modules/cli/pages/cbft-bleve-moss.adoc

```diff
@@ -0,0 +1,521 @@
+= cbft-bleve for Moss Indexes
+:description: pass:q[The `cbft-bleve` tool supports commands that return information on _Moss_ index partitions.]
+:page-topic-type: reference
```

Comment:
```suggestion
:page-topic-type: reference
:page-aliases: cbft-bleve-dump
```

Reply:
Hi @simon-dew the page alias "cbft-bleve-dump" that you mentioned is actually added to the new cbft-bleve.adoc file. This is the renamed file. So, there is no need to add that page alias to "cbft-bleve-moss.adoc" file.

Reply:
Ah sorry, I missed that!